test/CompanyService.Mappers.UnitTests/ResponseMappers/DepartmentResponseMapperTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using LT.DigitalOffice.CompanyService.Mappers.ResponsesMappers;
+using LT.DigitalOffice.CompanyService.Mappers.ResponsesMappers.Interfaces;
+using LT.DigitalOffice.CompanyService.Models.Db;
+using LT.DigitalOffice.CompanyService.Models.Dto.Models;
+using LT.DigitalOffice.UnitTestKernel;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.CompanyService.Mappers.UnitTests.ResponseMappers
+{
+    class DepartmentResponseMapperTests
+    {
+        private IDepartmentMapper _mapper;
+
+        private Department _expectedDepartment;
+        private DbDepartment _newDbDepartment;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            _mapper = new DepartmentMapper();
+
+            _newDbDepartment = new DbDepartment()
+            {
+                Id = Guid.NewGuid(),
+                Name = "Name",
+                Description = "TestDescription",
+                DirectorUserId = Guid.NewGuid(),
+                IsActive = true,
+                Users = null
+            };
+
+            _expectedDepartment = new Department
```

Comment:
I think you need to keep one style here and on 23 line

Reply:
I mean using ‘ () ‘

Reply:
Haven't noticed that, fixed